CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # edge-react-gui
 
+## 2.14.0 (2022-04-26)
```

Comment:
Add this tasks if they get verified:

https://github.com/EdgeApp/edge-react-gui/pull/3319
https://github.com/EdgeApp/edge-react-gui/pull/3317